docs/content/stable/deploy/kubernetes/single-zone/oss/helm-chart.md

```diff
@@ -422,6 +422,12 @@ helm repo update
 helm upgrade yb-demo yugabytedb/yugabyte --version {{<yb-version version="stable" format="short">}} --wait -n yb-demo
 ```
 
+Then finalize the upgrade as follows:
+
+```sh
+kubectl exec -it yb-master-0 -- /home/yugabyte/bin/yb-admin --master_addresses yb-master-0.yb-masters.default.svc.cluster.local:7100 finalize_upgrade
```

Comment:
```suggestion
kubectl exec -it yb-master-0 -- /home/yugabyte/bin/yb-admin --master_addresses yb-master-0.yb-masters.default.svc.cluster.local:7100 promote_auto_flags 
kubectl exec -it yb-master-0 -- /home/yugabyte/bin/yb-admin --master_addresses yb-master-0.yb-masters.default.svc.cluster.local:7100 upgrade_ysql
```

Reply:
Stable and previous version still need 2 steps. Only 2.25 has the new single combined command